pynestml/utils/ast_synapse_information_collector.py

```diff
@@ -0,0 +1,332 @@
+"""
```

Comment:
Does it make sense from an organisational perspective to have a subdirectory ``pynestml/utils/compartmental`` for these utils files?